theia-extensions/viewer-prototype/src/browser/trace-explorer/trace-explorer-widget.tsx

```diff
@@ -71,10 +73,14 @@ export class TraceExplorerWidget extends BaseWidget {
         signalManager().off(Signals.OPENED_TRACES_UPDATED, () => this.update());
```

Comment:
The listener does not get removed because it is not the same instance being passed that was added earlier.